src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -536,11 +536,11 @@ private static void FormatStandardAndErrorOutput(ITerminal terminal, string? sta
         terminal.SetColor(TerminalColor.DarkGray);
         terminal.Append(SingleIndentation);
         terminal.AppendLine(PlatformResources.StandardOutput);
-        string? standardOutputWithoutSpecialChars = NormalizeSpecialCharacters(standardOutput);
+        string? standardOutputWithoutSpecialChars = NormalizeSpecialCharacters(standardOutput, false);
```

Comment:
Some escaping of out/err was already done so I kept escaping of special chars but the "whitespace" ones. @nohwnd I don't know if we should simply escape nothing there as "special formatting" might be expected by users.